src/app/common/components/presentation/toolbar/toolbar.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-toolbar color="primary">
+  <span>{{title}}</span>
+</mat-toolbar>
```

Comment:
No newline at end of file. There are plug-ins for Visual Studio Code that will do this automatically for you.